general/telemetry.md

```diff
@@ -1,19 +1,40 @@
 # Telemetry in Zed
 
-Zed collects anonymous telemetry data to help the team understand how people are using the app. When we release the public beta, we'll provide the ability to opt out of telemetry, but during the alpha, it isn't configurable.
+Zed collects anonymous telemetry data to help the team understand how people are using the application and to see what sort of severe issues they are experiencing.
 
-The telemetry data is not shared with Zed's servers, but is sent directly to an analytics service called [Mixpanel](https://mixpanel.com), where it remains anonymous, and can't be related to specific Zed users. Events are reported over HTTPS, and requests are rate-limited to avoid using significant network bandwidth.
+## Types of Telemetry
 
-Currently, the events that are reported are:
+### Diagnostics
 
-* Opening the app
-* Signing in
-* Opening and saving files. File extensions (e.g. `.rs`, `.html`, `.txt`) are included in these events.
+Diagnostic events include debug information (stack traces) from crash reports.  Crash reports are sent to a cloud-monitoring service called [Datadog](https://www.datadoghq.com).  Within Datadog, we can visualize how frequently users are running into issues and assess the severity of the issue.  Having these crash reports sent automatically allows us to begin implementing fixes without the user needing to file a report in our issue tracker.  Being able to see various crashes plotted in Datadog also gives us an informal measurement of the stability of Zed.
 
-All events also include your OS version, Zed app version, and a timestamp.
+### Metrics
 
-## Auditing your Telemetry Events
+Zed also collects metric information based on user actions.  Metric events are reported over HTTPS, and requests are rate-limited to avoid using significant network bandwidth. The telemetry data is not shared with Zed's servers; it is sent directly to an analytics service called [Mixpanel](https://mixpanel.com), where it remains anonymous, and can't be related to specific Zed users.
 
-You can see the telemetry data that Zed has reported by running the command `zed: open telemetry log` from the command palette, or clicking `Help > View Telemetry Log` in the application menu.
+The following actions are captured:
+
+- Opening the app
+- Signing in
+- Opening and saving files. File extensions (e.g. `.rs`, `.html`, `.txt`) are included in these events; code is *never* included in these events.
+
+All `metrics` events also include your OS version, Zed app version, and a timestamp.
+
+#### Auditing your Metric Events
```

Comment:
Small suggestion to use #, ##, and ### for headings everywhere as those map 1:1 to GitBook's headings.

```suggestion
### Auditing your Metric Events
```

Reply:
Ah, so they don't have a `####`?  I used that only because this section only applies to the metric events.  I could just remove the header and work the sentence into the body of the previous `###` section.

Reply:
I believe it may render, but it will not be semantically significant in the context of GitBook.

Reply:
Ok, I just went ahead and removed it.  Appreciate your knowledge here on GitBook Petros.